### ISSUE TITLE
No issues notification is misleading

### ISSUE THREAD

**Issue by mtelgkamp** [cksource/ckeditor4-plugin-a11ychecker#264]
The message "**The document does not contain any accessibility issues.**" may be misleading, as there could be accessibility issues that cannot be checked automatically or ones that are not (yet) tested.
For example the document is validated fine when using an h3 to mark a complete paragraph "as bold text". The only requirement to pass is that the previous heading is an h2.
![image](https://user-images.githubusercontent.com/33117529/65303927-9fcf4000-db7f-11e9-9af5-ef2443769c73.png)

Additionally when the editor is used as editor for partial page content like it is in some CMS there could be problems in the interference with other page contents that cannot be checked within the editor.
